src/executables/unload_schema.c

```diff
@@ -1099,18 +1201,23 @@ extract_schema (extract_context & ctxt, print_output & schema_output_ctx)
 	}
     }
 
+  if (ctxt.is_dba_user == false && ctxt.is_dba_group_member == false)
+    {
+      filter_user_classes (&ctxt.classes, ctxt.login_user);
```

Comment:
Should we `ctxt.login_user` verify whether NULL or not?
I found `ctxt.classes` is checked. But `ctxt.login_user` is not found.